script/Arcueid/Day 10/10_00_ARC10_3_2.txt

```diff
@@ -832,7 +838,7 @@ Kagu Middle. That's an abbreviation of Kaguraoka Middle School that I went to.
 [sha:d20a08e0defe19a3153984a572a09f91c93cea62]{
 -- Page 227, Offset 24845.
 -- 「一身上の都合で休学中です。家にもいづらいので、ここでニートしています。
-"I'm taking time off school for personal reasons. It's tough staying at home as well, so I'm staying here as a NEET.
+"I'm taking time off school for personal reasons. It's tough staying at home though, so I'm staying here as a NEET.
```

Comment:
wow this guy is _LITERALLY ME_ 
![joker-thomas-wayne-reveal-1165610-1280x0](https://user-images.githubusercontent.com/97076550/159512704-50cdfb95-3a01-43ec-ac2c-df5bee50864f.jpeg)